lessons/python/scikit-learn/lesson.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: Machine Learning with scikit-learn
+visible: true
+tags:
+	- python
+	- intermediate
+---
+<!-- change visible to true if you want it on the site -->
+
+# Machine learning in Python
+
+ - **Authors**: Nil Sahin
+ - **Research field**: Computational Biology and Molecular Genetics
+ - **Lesson topic**: Machine Learning using Python's Sklearn packages
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/misc/scikit-learn>
```

Comment:
@nsahin I'll make it clearer. Change line 16, which is right now:

`- **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/misc/scikit-learn>`

to 

`- **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/python/scikit-learn>`

The only thing different is the `misc` -> `python` in the link.